Automattic/wp-calypso
# Jetpack Onboarding: Update remaining illustrations in JPO flow

Resolves #23269 

This PR updates five illustration assets in three of the steps 2, 3, and 4 of the Jetpack Onboarding (JPO) flow. The filenames remain the same.

Before/After:

Step 2 (Site type: personal or business):
<img width="748" alt="step2" src="https://user-images.githubusercontent.com/204742/37495999-ae9e29c0-2875-11e8-893f-c045ea48510a.png">

Step 3 (Homepage type step: news/updates or static):
<img width="740" alt="step3" src="https://user-images.githubusercontent.com/204742/37496150-6e236526-2876-11e8-9987-eeabf69c4591.png">

Step 4 (Contact form):
<img width="747" alt="step4" src="https://user-images.githubusercontent.com/204742/37496324-7e409130-2877-11e8-86b2-1053e02e75e6.png">



**Testing instructions:**
- Checkout this branch
- Pick a Jetpack site.
- Head to `/wp-admin/admin.php?page=jetpack&action=onboard&calypso_env=development` on the Jetpack site to start the onboarding flow.
- On Step 2, the site type step (personal or business), verify the illustration looks the same as the above "After" shot for Step 2.
- On Step 3, the homepage type step (news/updates or static), verify the illustration looks the same as the above "After" shot for Step 3.
- On Step 4, the contact form step, verify the illustration looks the same as the above "After" shot for Step 4.
